.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
   ],
   "git.branchProtectionPrompt": "alwaysCommitToNewBranch",
   "git.pullBeforeCheckout": true,
-  "git.mergeEditor": true,
+  "git.mergeEditor": false,
```

Comment:
see other PR. leave user to decide

Reply:
Done ab490ce7a0f1880066067af05f1f8ab8cdca9a65